### ISSUE TITLE
Change column names per r4ss updates 

### ISSUE THREAD

**Issue by e-perl-NOAA** [nmfs-fish-tools/SSMSE#216]
Column header updates in r4ss in PR [#945](https://github.com/r4ss/r4ss/pull/945) will require updates in SSMSE functions such as those with [FltSvy](https://github.com/search?q=repo%3Anmfs-fish-tools%2FSSMSE+FltSvy&type=code).

Additional changes in r4ss headers can be found in the [PR files changed](https://github.com/r4ss/r4ss/pull/945/files).

**Comment by k-doering-NOAA** [comment]
I'm currently trying to fix this.

I think there are 3 approaches possible
1) SSMSE only works with r4ss < 1.50.0 (before the breaking changes)
2) SSMSE only works with r4ss >= 1.50.0 (after the breaking changes)
3) SSMSE works with either version of r4ss, taking into account the different names.

I'm currently thinking 2) may be the best option moving forward, as maintaining both sets of column names will be cumbersome. It is fairly straightforward for users to update to a newer version of r4ss (a little more difficult to roll back to an older version), and the newer names in r4ss are more consistent and clear.

If anyone disagrees, please feel free to chime in!

**Comment by nathanvaughan-NOAA** [comment]
@k-doering-NOAA  I agree that option 2 is probably the best solution. We should also add a requirement in SSMSE for r4ss>=1.50.0 so that users get an alert if they have an incompatible r4ss version.

**Comment by k-doering-NOAA** [comment]
@nathanvaughan-NOAA thanks, I'm glad you agree!

I realized this is straightforward, but a little tricky because the SSMSE `sample_struct` object does not match with the r4ss names always, only sometimes, and I don't think we want to change the sample_struct names. Therefore, we'll need to find and replace *ONLY* the r4ss column names and not the original column names in `sample_struct`.

Alternatively, we could make the sample_struct names also match with the new r4ss names, which are more consistent and clear. This would be a breaking change in SSMSE, though.

**Comment by nathanvaughan-NOAA** [comment]
@k-doering-NOAA I'm busy at the Gulf SSC meeting for the next few days but I was thinking the same thing about the sample_struct naming issue. We have added a bunch of options to that for discards and custom MS inputs. Let me check in with @skylersagarese-NOAA, @CassidyPeterson-NOAA, @latreesedenson-NOAA, and @matthewsmith-NOAA when I get back for their thoughts. This could be a good opportunity to clean up name matches.

**Comment by e-perl-NOAA** [comment]
@nathanvaughan-NOAA, I just wanted to circle back to this to see if there is a particular solution that you would like to go with so that the capability to work with 3.30.21 can maybe move forward.

**Comment by nathanvaughan-NOAA** [comment]
Thanks @e-perl-NOAA  I'll check in with the rest of the SEFSC SSMSE folks again to see if we have an idea of when the best time would be to make the transition.

**Comment by k-doering-NOAA** [comment]
Resolve this within the next month

**Comment by kellijohnson-NOAA** [comment]
@k-doering-NOAA it would be nice for me if we could have a co-working session where we fix this in SSMSE and ss3sim at the same. If you are willing to do that, feel free to add something to my calendar at a time that is convenient for you.

**Comment by k-doering-NOAA** [comment]
That sounds great! I sent a meeting invite.

**Comment by k-doering-NOAA** [comment]
@kellijohnson-NOAA and I will finish changes in SSMSE and ss3sim by COB 5/19

**Comment by kellijohnson-NOAA** [comment]
@k-doering-NOAA the column names have also changed in `CPUEinfo` to `fleet units errtype SD_report`
